packages/ra-language-french/src/index.ts

```diff
@@ -47,12 +47,12 @@ const frenchMessages: TranslationMessages = {
         page: {
             create: 'Créer %{name}',
             dashboard: 'Tableau de bord',
-            edit: '%{name} #%{id}',
+            edit: '%{name} %{recordRepresentation}',
```

Comment:
This is a breaking change for all translation packages

Reply:
No it's not. Existing translations will still work as before. We still pass the `id`, the `recordRepresentation` is passed in addition to other message args. The new translation takes advantage of it.